src/terminal/terminal.rs

```diff
@@ -568,6 +568,65 @@ where
     }
 }
 
+#[cfg(feature = "crossterm")]
+impl Terminal<CrosstermBackend<Stdout>> {
+    /// Initializes the terminal with reasonable defaults for most applications.
+    ///
+    /// The Terminal is initialized with the following settings:
+    ///
+    /// - Stdout as the output
+    /// - Crossterm backend
+    /// - Raw mode enabled
+    /// - Alternate screen buffer enabled
+    /// - A panic hook that restores the terminal before panicking
+    ///
+    /// For more control over the terminal initialization, use [`Terminal::new`] or
+    /// [`Terminal::with_options`].
+    ///
+    /// # Examples
+    ///
+    /// ```rust,no_run
+    /// let terminal = ratatui::Terminal::init()?;
+    /// # std::io::Result::Ok(())
+    /// ```
+    pub fn init() -> io::Result<Terminal<CrosstermBackend<Stdout>>> {
+        Self::set_panic_hook();
+        crossterm::terminal::enable_raw_mode()?;
+        crossterm::execute!(std::io::stdout(), crossterm::terminal::EnterAlternateScreen)?;
+        let backend = CrosstermBackend::new(std::io::stdout());
+        Ok(Terminal::new(backend)?)
+    }
+
+    /// Restores the terminal to its original state.
```

Comment:
It should be a bit more clear that it is meant to be used with the `crossterm` based init method.

```suggestion
    /// Restores the crossterm terminal on stdout to its original state.
```